transport/src/main/java/io/netty/bootstrap/ServerBootstrap.java

```diff
@@ -183,8 +287,8 @@ public void initChannel(Channel ch) throws Exception {
                 if (handler != null) {
                     pipeline.addLast(handler);
```

Comment:
You can collect them fairly easily because you specified them when you build a `ServerBootstrap`.